src/test/groovy/edu/ie3/datamodel/io/naming/FileNamingStrategyTest.groovy

```diff
@@ -98,28 +99,28 @@ class FileNamingStrategyTest extends Specification {
 
     then:
     res.present
-    res.get() == expectedString
+    res.get() == expectedPath
 
     where:
-    modelClass               || expectedString
-    LoadResult               || "test_grid" + File.separator + "results" + File.separator + "participants"
-    FixedFeedInResult        || "test_grid" + File.separator + "results" + File.separator + "participants"
-    BmResult                 || "test_grid" + File.separator + "results" + File.separator + "participants"
-    PvResult                 || "test_grid" + File.separator + "results" + File.separator + "participants"
-    ChpResult                || "test_grid" + File.separator + "results" + File.separator + "participants"
-    WecResult                || "test_grid" + File.separator + "results" + File.separator + "participants"
-    StorageResult            || "test_grid" + File.separator + "results" + File.separator + "participants"
-    EvcsResult               || "test_grid" + File.separator + "results" + File.separator + "participants"
-    EvResult                 || "test_grid" + File.separator + "results" + File.separator + "participants"
-    EmResult                 || "test_grid" + File.separator + "results" + File.separator + "participants"
-    FlexOptionsResult        || "test_grid" + File.separator + "results" + File.separator + "participants"
-    Transformer2WResult      || "test_grid" + File.separator + "results" + File.separator + "grid"
-    Transformer3WResult      || "test_grid" + File.separator + "results" + File.separator + "grid"
-    LineResult               || "test_grid" + File.separator + "results" + File.separator + "grid"
-    SwitchResult             || "test_grid" + File.separator + "results" + File.separator + "grid"
-    NodeResult               || "test_grid" + File.separator + "results" + File.separator + "grid"
-    CylindricalStorageResult || "test_grid" + File.separator + "results" + File.separator + "thermal"
-    ThermalHouseResult       || "test_grid" + File.separator + "results" + File.separator + "thermal"
+    modelClass               || expectedPath
+    LoadResult               || Paths.get("test_grid", "results", "participants")
```

Comment:
You could replace all `Paths.get` in here with `Path.of`, but it's not super important